go.mod

```diff
@@ -1072,3 +1072,5 @@ replace (
 
 // Prevent a false-positive detection by the Google and Ikarus security vendors on VirusTotal
 exclude go.opentelemetry.io/proto/otlp v1.1.0
+
+replace github.com/google/gopacket v1.1.19 => github.com/DataDog/gopacket v0.0.0-20240626205202-4ac4cee31f14
```

Comment:
Is the change intentional? do we have a reason why we're not using an official version?

Reply:
Yes, for this change: https://github.com/google/gopacket/pull/1168

I am not sure that PR is going to get merged just based on the history of pull requests on that repo.

Reply:
Why is this a replace? You should be able to change the `require` line.

Reply:
The code references `github.com/google/gopacket`